packages/atomic/src/locales.json

```diff
@@ -700,32 +700,32 @@
     "fr": "Segments d'URI réduits"
   },
   "disconnected": {
-    "en": "Could not connect.",
-    "fr": "Impossible de se connecter."
+    "en": "No Internet access.",
+    "fr": "Pas d'accès internet."
   },
   "check-your-connection": {
     "en": "Your query couldn't be sent. Verify your internet connection.",
     "fr": "Votre requête n'a pas plus être envoyée. Peut-être que vous avez perdu votre connexion internet."
   },
   "no-endpoints": {
-    "en": "The Coveo Organization has no registered endpoints.",
-    "fr": "L'organisation Coveo n'a pas de point de terminaison unique."
+    "en": "Your organization {{org}} has no available content.",
+    "fr": "Votre organisation {{org}} n'a pas contenu disponible."
   },
   "invalid-token": {
-    "en": "The token is invalid.",
-    "fr": "Le jeton d'identification est invalide."
+    "en": "Ensure that the token is valid.",
+    "fr": "Faites en sorte que le jeton d'identification soit invalide."
   },
   "add-sources": {
-    "en": "You will need to add sources in your index, or wait for the created sources to finish indexing.",
+    "en": "Add content sources or wait for your newly created sources to finish indexing.",
     "fr": "Vous devrez ajouter des sources à votre index, ou attendre la fin de l'indexation des sources créées."
   },
   "coveo-online-help": {
     "en": "Coveo Online Help",
     "fr": "Aide en ligne Coveo"
   },
   "cannot-access": {
-    "en": "The Coveo Organization cannot be accessed.",
-    "fr": "L'organisation Coveo est inaccessible."
+    "en": "Your organization {{org}} cannot be accessed.",
+    "fr": "Votre organisation {{org}} est inaccessible."
```

Comment:
If this message is displayed when [a token is invalid](https://github.com/coveo/ui-kit/pull/1278/files#diff-27c2ff859d0380a8f543ec7d7ee7e69b8069da35daa341086a9b14feb236fdd6R102), it would be helpful to explain this in the message so that a support agent or developer can quickly understand why, e.g.

"Your organization {{org}} cannot be accessed because the access token is invalid."

Reply:
Description:

```
"invalid-token": {
    "en": "Ensure that the token is valid.",
    "fr": "Faites en sorte que le jeton d'identification soit invalide."
  },
```